Lesson_2_Python_II/Lesson_2_clean.ipynb

```diff
@@ -6,49 +6,34 @@
    "source": [
```

Comment:
<p>Lab:</p><p>-did you want to delete the statistic staff? seems pretty easy to figure out and they can start getting used to those terms</p><p>-on the last part to the this lab, maybe say print the number of players that are taller than 73 in</p><p>-spelling: variable, for loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
<p>I did delete it because we did not introduce these methods on the lesson. If I was to ask them to "perform statistical analysis" they would not know what we are referring to... what I am going to do is to add it back to the end of the lab. I will be adding details for them to know which specific metrics we are looking for.</p>